### DISCUSSION TITLE
latest/utils/image/

### DISCUSSION THREAD

**Discussion by giscus** [roboflow/supervision#793] [Docs]
# utils/image/

A set of easy-to-use utilities that will come in handy in any computer vision project.

https://supervision.roboflow.com/utils/image/

**Comment by GodLordGee** [comment]
The examples in Crop part,the last line 'sink.save_image(image=image)' , Should be modified as ‘sink.save_image(image=cropped_image)’,otherwise, you'll get the same picture.LOL😄

**Reply by SkalskiP** [reply]
Hi @GodLordGee 👋🏻 Thanks for letting us know. I just fixed it: https://github.com/roboflow/supervision/pull/797.
